query-engine/core/src/schema_builder/input_types/fields/data_input_mapper/update.rs

```diff
@@ -203,3 +196,120 @@ fn update_operations_object_type(
 
     Arc::downgrade(&obj)
 }
+
+// Builds a `set` input object type. Should be used for the shorthand syntax and within the update envelope type.
+fn composite_set_update_object_type(ctx: &mut BuilderContext, cf: &CompositeFieldRef) -> InputObjectTypeWeakRef {
```

Comment:
I'm pretty sure this is unnecessary. How would this ever differ from the set input that is created for the create operations?

Reply:
s t u p i d 🤦‍♂️